packages/binding-websockets/src/ws-server.ts

```diff
@@ -179,6 +179,25 @@ export default class WebSocketServer implements ProtocolServer {
                     encodeURIComponent(propertyName);
                 const property = thing.properties[propertyName];
 
+                // Populate forms related to the property
+                for (const address of Helpers.getAddresses()) {
+                    const href = this.scheme + "://" + address + ":" + this.getPort() + path;
```

Comment:
You could also use a string literal for this line:

```suggestion
                    const href = `${this.scheme}://${address}:${this.getPort()}${path}`;
```

Not sure if it improves readability, though.